src/Command/ForceRunCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Scheduler\Command;
+
+use Contributte\Scheduler\IScheduler;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ForceRunCommand extends Command
+{
+
+	/** @var string */
+	protected static $defaultName = 'scheduler:forceRun';
```

Comment:
I prefer snake_case.

Reply:
Fixed

Reply:
Oh shit, I ment camel-case. Sorry. Too early for me. :-D

`scheduler:force-run`